frontend/src/general/userForm/__tests__/FormConsole.test.jsx

```diff
@@ -0,0 +1,160 @@
+import {
+  afterEach,
+  beforeEach,
+  describe,
+  expect,
+  it,
+  jest
+} from "@jest/globals";
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { Button, Container, TextField } from "@material-ui/core";
+import FormConsole from "../FormConsole";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<FormConsole/> class component", () => {
+  let wrapper;
+  const mockHandleSubmit = jest.fn();
+  const dummyValues = {
+    handleSubmit: mockHandleSubmit,
+    buttonName: "Create"
+  };
+
+  beforeEach(() => {
+    wrapper = Enzyme.shallow(
+      <FormConsole
+        handleSubmit={dummyValues.handleSubmit}
+        buttonName={dummyValues.buttonName}
+        isValidate
+        isCreateUser
+      />
+    );
+  });
+
+  afterEach(() => {
+    wrapper.unmount();
+  });
+
+  describe("render()", () => {
+    it("should have 1 <Container/>, 2 <TextFields/> and 1 <Button/>", () => {
+      expect(wrapper.find(Container)).toHaveLength(1);
+      expect(wrapper.find(TextField)).toHaveLength(2);
+      expect(wrapper.find(Button)).toHaveLength(1);
+    });
+
+    it("contains the props buttonName as a child of <Button/>", () => {
+      expect(wrapper.find(Button).children().text()).toEqual(
+        dummyValues.buttonName
+      );
+    });
+
+    it("if isValidate prop is true, it should have the form's noValidate to true", () => {
+      const wrapperIsValidate = Enzyme.shallow(
+        <FormConsole
+          handleSubmit={dummyValues.handleSubmit}
+          buttonName={dummyValues.buttonName}
+          isValidate
+        />
+      );
```

Comment:
is this new shallow really necessary? the one in your `beforeEach` also has `isValidate` so this feels redundant

Reply:
I think I forgot to remove isCreate and isValidate from the wrapper found in beforeEach() which is why I created new shallow renders to test those components passed with those props in isolation. I'll have to modify this test to reflect that.

Reply:
ah if you're removing those from the first shallow then are you also going to add checks for `isValidate` = false and `isCreateUser` = false? i don't think you currently have those test cases

Reply:
Done isValidate = false, the other one isn't applicable anymore since I've refactored isCreateUser to instead be error props related to the password TextField in FormConsole.